modules/user-guide/pages/setting-up-the-github-oauth-app.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+:description: Setting up the GitHub OAuth App
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, private-github, private, github, github-repo, github-repository
+:navtitle: Setting up the GitHub OAuth App
+// :page-aliases:
+
+[id="setting-up-the-github-oauth-app_{context}"]
+= Setting up the GitHub OAuth App
+
+To be able to access a private Git repository that is hosted on GitHub, set up a GitHub OAuth App as follows:
+
+.Prerequisites
+
+* You are logged in to GitHub.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Go to link:https://github.com/settings/applications/new[].
+//The long version: Click your GitHub avatar *>* *Settings* *>* *Developer settings* *>* *OAuth Apps* *>* *Register a new application*. max-cx
+
+. Enter *{prod-short}* as the *Application name*.
+
+. Enter `{prod-url}/` as the *Homepage URL*.
+
+. Enter `{prod-url}/api/oauth/callback` as the *Authorization callback URL*.
+
+. Click *Register application*.
+
+. Click *Generate new client secret*.
+
+. Save and then encode to Base64 the *GitHub OAuth Client ID* like so:
++
+----
+$ echo -n '__<github_oauth_client_id>__' | base64
```

Comment:
```suggestion
$ echo -n '<github_oauth_client_id>' | base64
```

Since these lines are not italicized:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/83611742/160716029-e6c405b4-de4c-42ae-8218-c6009fcc3250.png">